modules/drivers/numpy/wscript

```diff
@@ -34,10 +34,9 @@ def configure(conf):
                        includes=inc_dir,
                        libpath=lib_dir,
                        msg='Checking for library numpy' + warningmessage, 
-                       mandatory=False) 
-            
+                       mandatory=True)
```

Comment:
Hmm... When configuring and looking for `numpy/arrayobject.h` waf never finds anything for me on miinfcompute01. With mandatory=False configuration completes successfully and compilation fails when it can't find `numpy/arrayobject.h`. With mandatory=True configuration completes successfully and the numpyutils-c++ task never gets created. But when looking for "numpy/numpyconfig.h" waf _does_ find it and successfully compiles (regardless of mandatory setting)?